src/styles.scss

```diff
@@ -4,6 +4,10 @@
   color: #3b3b3b;
 }
 
+html, body {
+  overflow-x: hidden;
```

Comment:
Try not to set that kind of agressive styles on root elements like html or body. Just use it directly where needed. If nedded.